pkg/frameworks/wordpress/testdata/version.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * WordPress Version
+ *
+ * Contains version information for the current WordPress release.
+ *
+ * @package WordPress
+ * @since 1.1.0
+ */
+
+ // $wp_version = '4.5.6-alpha';
+
+ /* 
+    $wp_version = '4.5.6-alpha';
```

Comment:
This case will fail.

```suggestion
$wp_version = '4.5.6-alpha';
```